@kiva/kv-components/vue/KvRadio.vue

```diff
@@ -8,14 +8,13 @@
 			<input
 				v-bind="$attrs"
 				:id="uuid"
-				ref="radioRef"
+				:ref="radioRef"
```

Comment:
I think this also needs to change back to `ref="radioRef"`

Reply:
Ready!